libr/anal/esil.c

```diff
@@ -2568,17 +2567,20 @@ static int esil_num(RAnalEsil *esil) {
 
 /* duplicate the last element in the stack */
 static int esil_dup(RAnalEsil *esil) {
-	if (!esil || !esil->stack || esil->stackptr < 1 || esil->stackptr > (esil->stacksize - 1))
+	if (!esil || !esil->stack || esil->stackptr < 1 || esil->stackptr > (esil->stacksize - 1)) {
```

Comment:
Maybe make it multiline?

Reply:
we should probebly use an RInterval and the whole thing will be simpler